package.json

```diff
@@ -39,12 +39,14 @@
     "github": "https://github.com/canjs/canjs"
   },
   "dependencies": {
+    "bit-docs-generate-html": "^0.6.0",
```

Comment:
this is already listed in the bit-docs dependencies so this wouldn't need to be here